manifests/application/migrator/log4j.pp

```diff
@@ -0,0 +1,26 @@
+class lightblue::application::migrator::log4j(
+  $log_level = 'INFO',
+  $log_file_name = 'migrator.log',
+  $log_max_file_size = '1MB',
+  $log_max_backups_to_keep = '5',
+  $log_pattern = '%d [%t] %c %p - %m%n',
+  $config_dir,
+  $log_dir,
+  $owner,
+  $group,
+  $service_name
```

Comment:
Regarding defaults for lightblue::application::migrator::log4j I recommend:
* $log_max_file_size = '10MB'
* $log_max_backups_to_keep = '50'
* $log_pattern = '%d [%t] %-5p [%c] %m%n'

These are consistent with other logging we have for other systems.